practice-typescript/src/scripts/constants/message.ts

```diff
@@ -1,7 +1,7 @@
 enum MESSAGE_ERROR {
   EMAIL = 'Please enter a valid email address',
-  FIRST_NAME = 'Please enter your first name',
-  LAST_NAME = 'Please enter your last name',
+  FIRST_NAME = 'Please enter your first name with a maximum of 20 characters',
```

Comment:
Separate error messages for each case:
- Empty: => Please enter your first name
- Contain special character: => Your first name should not include a special character
- Greater than length: => First name cannot exceed 12 characters
Or Don't allow user input to exceed the length limit